opslevel/datasource_opslevel_service.go

```diff
@@ -153,6 +180,25 @@ func datasourceServiceRead(d *schema.ResourceData, client *opslevel.Client) erro
 		return err
 	}
 
+	properties, err := resource.GetProperties(client, nil)
+	if err != nil {
+		return err
+	}
+	// log warnings for any validation errors rather than adding them to state
+	for _, property := range properties.Nodes {
+		for _, validationErr := range property.ValidationErrors {
+			log.Warn().Msgf("service '%s' property '%s' has a validation error\n\tmessage=\"%s\" path=[%s]",
```

Comment:
The log message output looks like this if a validation error is present:

```
2023-12-22T14:00:05.512-0500 [DEBUG] provider.terraform-provider-opslevel_1703271533.0.0: {"level":"warn","time":"2023-12-22T14:00:05-05:00","message":"service 'Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS85MTQyOQ' property 'Z2lkOi8vb3BzbGV2ZWwvUHJvcGVydGllczo6RGVmaW5pdGlvbi8xODA' has a validation error\n\tmessage=\"Value doesn't satisfy the schema: root value 'invalid' is not one of: red, green, blue, yellow, purple\" path=[value]"}
```

However `TF_LOG=DEBUG` needs to be set. Couldn't find any way to get the provider to send a message that doesn't get filtered by terraform. 

[logging docs](https://developer.hashicorp.com/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-logging)

Reply:
I think thats because we need to use the `tflog` package - lets cut a ticket to investigate this for the future.  We should know how to do this so we can properly communicate things like this to the end user.

Reply:
https://github.com/OpsLevel/team-platform/issues/162